painttiming.bs

```diff
@@ -177,8 +188,8 @@ The {{PerformancePaintTiming}} interface {#sec-PerformancePaintTiming}
 * The {{PerformanceEntry/startTime}} attribute's getter must return a {{DOMHighResTimeStamp}} of when the paint occured.
 * The {{PerformanceEntry/duration}} attribute's getter must return 0.
 
-NOTE: A user agent implementing {{PerformancePaintTiming}} would need to include <code>"paint"</code> in {{PerformanceObserver/supportedEntryTypes}} for {{Window}} contexts.
-This allows developers to detect support for paint timing.
+NOTE: A user agent implementing {{PerformancePaintTiming}} would need to include <code>"paint"</code> in {{PerformanceObserver/supportedEntryTypes}} of a [=global object=], where its [=responsible browsing context=] is [=paint-timing eligible=].
```

Comment:
nit: s/where/whose. Also, the `responsible browsing context` is not of the `global object` but of the `relevant settings object`, which BTW can be for any platform object (including Windows).